docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -119,6 +120,7 @@ public DockerTemplateBase(String image,
                               Integer cpuShares,
                               String bindPorts,
                               boolean bindAllPorts,
+                              boolean bindSshPortLocalhost,
```

Comment:
I'd prefer a more flexible binding definition, falling back to `0.0.0.0` when not set, but offering more flexibility for various use-cases than just "limit to localhost"